Spring_Server/gs-uploading-files-complete/upload-dir/build.gradle

```diff
@@ -55,3 +57,85 @@ tasks.withType(Test) {
         }
     }
 }
+
```

Comment:
Commented code?

Reply:
That is there for testing purposes for the build.gradle I could change it if you want